### DISCUSSION TITLE
HW 01, Problems Part 3, Question 3

### DISCUSSION THREAD

**Discussion by feliciones** [cooperrc/computational-mechanics#32] [Q&A]
Does this graph look correct for the final question on HW 1?

![image](https://user-images.githubusercontent.com/106564968/172071089-804770f6-4ab1-44ab-acec-3e9cdd327967.png)

**Comment by deena101** [comment]
What values did you use for the array?

**Comment by cooperrc** [comment]
There should be a decreasing error. 

doublecheck that you're comparing :apple: <-> :apple:

`np.exp(1) == 2.718281828459045...` **constant**

`exptaylor(1, 2) == 2.5` 

`exptaylor(1, 10) == 2.7182818011463845`
